board/src/teensy4.rs

```diff
@@ -35,3 +35,9 @@ pub fn prepare(timer_delay_microseconds: u32) -> Option<crate::Resources> {
         pit,
     })
 }
+
+#[cfg(feature = "__dcd")]
+#[link_section = ".dcd"]
+#[no_mangle]
+#[used]
+pub static DEVICE_CONFIGURATION_DATA: [u8; 7] = [0xde, 0xad, 0xbe, 0xef, 0xca, 0xfe, 0xa5];
```

Comment:
I just realized that valid DCD data is aligned to 4 bytes, so I did not re-align in the linker script, nor did I validate this property. In retrospect both should be done.